Add changes for multiple signing keys (#8243)
opensearch-project/documentation-website: _security/authentication-backends/jwt.md

Name | Description
:--- | :---

> `signing_key` | The signing key(s) to use when verifying the token. If you use a symmetric key algorithm, it is the base64-encoded shared secret. If you use an asymmetric algorithm, the algorithm contains the public key. To pass multiple keys, use a comma-separated list or enumerate the keys.

Reviewer: ```suggestion
`signing_key` | The signing key(s) used to verify the token. If you use a symmetric key algorithm, this is the Base64-encoded shared secret. If you use an asymmetric algorithm, the algorithm contains the public key. To pass multiple keys, use a comma-separated list or enumerate the keys.
```
